x-pack/plugins/global_search/server/services/context.ts

```diff
@@ -30,6 +31,7 @@ export const getContextFactory = (coreStart: CoreStart) => (
       uiSettings: {
         client: coreStart.uiSettings.asScopedToClient(soClient),
       },
+      capabilities: from(coreStart.capabilities.resolveCapabilities(request)),
```

Comment:
The whole GS internal API being based on observable, it was tedious to consume a promise from the results provider, so I exposed an observable instead. It's also more consistent with the provider internal API.